homeassistant/components/aurora_abb_powerone/sensor.py

```diff
@@ -75,6 +74,8 @@ async def async_setup_entry(
 class AuroraSensor(AuroraEntity, SensorEntity):
     """Representation of a Sensor on a Aurora ABB PowerOne Solar inverter."""
 
+    _attr_has_entity_name = True
```

Comment:
Does this mean that the default name and entity id will change if someone sets up a new device or removes and re-adds a device?

Reply:
https://developers.home-assistant.io/docs/core/entity#has_entity_name-true-mandatory-for-new-integrations

Reply:
Ok thank you.